hermes/parsers/hermes_message_parser.py

```diff
@@ -24,22 +24,16 @@ def parse_message(self, message):
         self.link_message(message)
         return True
 
-    def find_and_link_targets(self, structure, message):
-        if isinstance(structure, dict):
-            if 'target_name' in structure and 'ra' in structure and 'dec' in structure:
-                    target, _ = Target.objects.get_or_create(name=structure['target_name'], coordinate=Point(float(structure['ra']), float(structure['dec']), srid=4035))
-                    if not target.messages.contains(message):
-                        target.messages.add(message)
-                        target.save()
-            else:
-                for value in structure.values():
-                    if isinstance(value, (list, dict)):
-                        self.find_and_link_targets(value, message)
-        elif isinstance(structure, list):
-            for value in structure:
-                if isinstance(value, (list, dict)):
-                    self.find_and_link_targets(value, message)
-
+    def link_targets(self, data, message):
+        for target_details in data.get('targets', []):
+            if target_details.get('ra') and target_details.get('dec'):
+                target, _ = Target.objects.get_or_create(
```

Comment:
If a target is ingested and the RA or Dec have different precision than an existing target, this will create a new target with the same name and essentially the same coordinates, correct?

Reply:
yes, I believe so